services/files/run_push.go

```diff
@@ -32,10 +32,25 @@ type PushParams struct {
 	FileType    string
 	Directives  map[string]string
 	JobIDOrName string
+	NoJob       bool
+}
+
+// Validate checks that the PushParams are valid
+func (p PushParams) Validate() error {
+	if p.JobIDOrName != "" && p.NoJob {
+		return ErrInvalidParamCombination([]string{"job", "nojob"})
```

Comment:
There are more conflicting params. Pls check the jira. You can't specify a locale or autoauthorize.